MATLAB/DFA/src/EDA/exper_data.m

```diff
@@ -0,0 +1,131 @@
+%{ 
+Experimental Data
+Author: Surabhi Joshi
+
+Table of Contents
+Experimental Data
+Summary
+Create tables for averages and standard deviations. 
+Create a corelation matrix and a following heatmap
+Create the correlation matrix, CV 
+Create heatmaps to visually understand the standard deviations and averages. 
+Scatterplots of the mean and standard deviations
+Hisograms of the mean and standard deviations
+
+
+
+Summary
+This script reads in some experimental data and performs exploratory initial statistical analysis to understand how each condition affects the metabolomics before running DFA. 
+% download the experimental data and create two tables, one for each
+% cancer cell line TU8902 and HCT116.
+%}
+
+experimental_data_TU8902 = readtable('Barb Nelson Pos Table-20180410 - HL.xlsx', ...
+'Sheet', 'TU8902');
+experimental_data_HCT116 = readtable('Barb Nelson Pos Table-20180410 - HL.xlsx', ... 
+ 'Sheet',"HCT116");
+% delete row 2 and column 2 from both tables
+experimental_data_TU8902.transition = [];
+experimental_data_HCT116.transition = [];
+
+
+%%Create tables for averages and standard deviations. 
+%%This here is going to take the averages and standard deviations from the original experimental data and fill two new tables with their values. Any NaN will be turned to 0 prior to finding the mean and standard deviations. 
+Tabl_average_TU8902 = experimental_data_TU8902;
+Tabl_std_TU8902 = experimental_data_TU8902;
+Tabl_average_HCT116 = experimental_data_HCT116;
+Tabl_std_HCT116 = experimental_data_HCT116;
+
+%fill the tables with averages and standard deviations and then delete the
+%the last two trials for each condition
+for n = 2:13
+    Tabl_average_TU8902(:,(n)) = num2cell(nanmean(Tabl_average_TU8902{:,n:n+2},2));
+    Tabl_std_TU8902(:,(n)) = num2cell(nanstd(Tabl_std_TU8902{:,n:n+2},[ ],2));
+    Tabl_average_HCT116(:,(n)) = num2cell(nanmean(Tabl_average_HCT116{:,n:n+2},2));
+    Tabl_std_HCT116(:,(n)) = num2cell(nanstd(Tabl_std_HCT116{:,n:n+2},[ ],2));
+    
+    Tabl_average_TU8902(:,(n+2)) = [];
+    Tabl_std_TU8902(:,(n+2)) = [];
+    Tabl_average_TU8902(:,(n+1)) = [];
+    Tabl_std_TU8902(:,(n+1)) = [];
+    
+    Tabl_average_HCT116(:,(n+2)) = [];
+    Tabl_std_HCT116(:,(n+2)) = [];
+    Tabl_average_HCT116(:,(n+1)) = [];
+    Tabl_std_HCT116(:,(n+1)) = [];

```

Comment:
I don't think this for loop computes the values correctly still. 

If your columns are:
cond1 replicate 1 | cond1 replicate 2 | cond1 replicate 3 | cond2 replicate 1 ...

your for loop computes this:
i = 1: mean(cond1 replicate 1 | cond1 replicate 2 | cond1 replicate 3 | )
i = 2: mean(cond1 replicate 2 | cond1 replicate 3 | cond2 replicate 1 | )

and so on.